prototype/p2p/messenger_test.go

```diff
@@ -0,0 +1,78 @@
+package p2p
+
+import (
+	"pocket/p2p/types"
+	"testing"
+)
+
+func TestMessager_Protobuff(t *testing.T) {
```

Comment:
Messager -> Messenger?
Protobuff -> Protobuf?

Why the `_` in the test name?

Reply:
To allow for targeted testing using `go test -run TestMessenger` or `go test -run TestNetwork`